bot/exts/moderation/infraction/_utils.py

```diff
@@ -269,3 +272,43 @@ async def send_private_embed(user: MemberOrUser, embed: discord.Embed) -> bool:
             "The user either could not be retrieved or probably disabled their DMs."
         )
         return False
+
+
+class StaffBanConfirmationView(interactions.ViewWithUserAndRoleCheck):
+    """The confirmation view that is sent when a moderator attempts to ban a staff member."""
+
+    def __init__(
+        self,
+        *,
+        allowed_users: t.Sequence[int],
+        allowed_roles: t.Sequence[int],
+        timeout: t.Optional[float] = 180,
+        message: t.Optional[Message] = None
+    ) -> None:
+        super().__init__(
+            allowed_users=allowed_users,
+            allowed_roles=allowed_roles,
+            timeout=timeout,
+            message=message
+        )
+
+        self.value: t.Optional[bool] = None
+
+    async def send(self, channel: TextChannel, *, message_content: str) -> None:
```

Comment:
```suggestion
    async def send(self, channel: TextChannel, message_content: str) -> None:
```
In my opinion `message_content` doesn't need to be keyword
We can see what other people think as well